### ISSUE TITLE
forge command runs without package naming

### ISSUE THREAD

**Issue by neija2611** [poseidon-framework/poseidon-hs#154]
While using the "forge" command I didn't specify a name for the new created Poseidon package. No error occurred and the files (.bim, .fam, .bed, .janno, .bib) were created anyhow, but without being named. The package was called "" in the .yml file.

**Comment by nevrome** [comment]
Thanks for reporting that - it should indeed be impossible to create unnamed packages :+1:

**Comment by esalmela** [comment]
I stumbled upon the same issue, but in a slightly different fashion: in trident forge, I did specify a name for the new package, with full path and everything, but erroneously with a '/' in the end. This produced the new folder I asked for, and an unnamed package within it as above (iirc).

Fwiw, I wonder what would have happened, had I similarly typoed an earlier trident init command. A symmetric possibility of error may exist there.
